assets/js/script.js

```diff
@@ -0,0 +1,209 @@
+const currentWeatherContainer = $("#current-weather-container");
+const forecastWeatherContainer = $("#forecast-container");
+
+const API_KEY = "708fa10260c22a09e8551c978be4e26d";
+
+const getCurrentData = function (name, forecastData) {
+  return {
+    name: name,
+    temperature: forecastData.current.temp,
+    wind: forecastData.current.wind_speed,
+    humidity: forecastData.current.humidity,
+    uvi: forecastData.current.uvi,
+    date: getFormattedDate(forecastData.current.dt),
+    iconCode: forecastData.current.weather[0].icon,
+  };
+};
+
+// add bulma.io
+const getUVIClassName = function (uvi) {
+  if (uvi >= 0 && uvi < 3) {
+    return "button is-success";
+  } else if (uvi >= 3 && uvi < 6) {
+    return "button is-warning";
+  } else if (uvi >= 6 && uvi < 8) {
+    return "button is-danger";
+  } else {
+    return "button is-black";
+  }
+};
+
+const setCitiesInLS = function (cityName) {
+  // get cities from LS
+  const cities = JSON.parse(localStorage.getItem("recentCities")) ?? [];
+
+  // if city does not exist
+  if (!cities.includes(cityName)) {
+    // insert cityName in cities
+    cities.push(cityName);
+
+    // set cities in LS
+    localStorage.setItem("recentCities", JSON.stringify(cities));
+  }
+};
+
+const getFormattedDate = function (unixTimestamp) {
+  return moment.unix(unixTimestamp).format("ddd DD/MM/YYYY");
+};
+
+const getIconCode = function () {
```

Comment:
This function doesn't seem to do anything and it is not called anywhere. Remember to remove any code that you do not need.